docs/docs/explanations/advanced-configuration.md

```diff
@@ -0,0 +1,836 @@
+want to increase the resource levels associated with the Dask Scheduler.
+
+```yaml
+dask_worker:
+    "Huge Worker":
+      worker_cores_limit: 7
+      worker_cores: 6
+      worker_memory_limit: 30G
+      worker_memory: 28G
+      scheduler_cores_limit: 7
+      scheduler_cores: 6
+      scheduler_memory_limit: 30G
+      scheduler_memory: 28G
+```
+
+### JupyterLab Profile Node Selectors
+
+A common operation is to target jupyterlab profiles to specific node labels. In order to target a specific node groups add the following. This example shows a GKE node groups with
+name `user-large`. Other cloud providers will have different node labels.
+
+```yaml
+profiles:
+  jupyterlab:
+    - display_name: Small Instance
+      ...
+      kubespawner_override:
+        ...
+        node_selector:
+          "cloud.google.com/gke-nodepool": "user-large"
+        ...
+```
+
+### Specifying GPU/Accelerator Requirements
+
+If you want to ensure that you have GPU resources use the following annotations.
+
+```yaml
+profiles:
+  jupyterlab:
+    - display_name: Small Instance
+      ...
+      kubespawner_override:
+        ...
+        extra_resource_limits:
+          nvidia.com/gpu: 1
+        ...
+```
+
+## Customizing JupyterHub theme
+
+JupyterHub can be customized since QHub uses [Quansight/qhub-jupyterhub-theme](https://github.com/quansight/qhub-jupyterhub-theme). Available theme options.
+
+
+```yaml
+theme:
+  jupyterhub:
+    hub_title: QHub - thisisatest
+    hub_subtitle: Autoscaling Compute Environment
+    welcome: |
+      Welcome to jupyter.github-actions.qhub.dev. It's maintained by <a href="http://quansight.com">Quansight
+      staff</a>. The hub's configuration is stored in a github repository based on
+      <a href="https://github.com/Quansight/qhub/">https://github.com/Quansight/qhub/</a>.
+      To provide feedback and report any technical problems, please use the
+      <a href="https://github.com/Quansight/qhub/issues">github issue tracker</a>.
+    logo: /hub/custom/images/jupyter_qhub_logo.svg
+    primary_color: '#4f4173'
+    secondary_color: '#957da6'
+    accent_color: '#32C574'
+    text_color: '#111111'
+    h1_color: '#652e8e'
+    h2_color: '#652e8e'
+```
+
+:::note
+To properly update the image logo, you will must ensure that the provided logo field contains an accessible **URL** to the logo.
+:::
+
+Its also possible to display the current version of qhub by using the `display_version: 'True'` in the above `theme.jupyterhub` configuration. If no extra information is passed,
+the displayed version will be the same as `qhub_version`, an overwrite can be done by passing the `version: v.a.b.c` key as well.
+
+## Filesystem environments
+
+QHub is experimenting with a new way of distributing environments using [conda-store](https://github.com/quansight/conda-store). Please expect this environment distribution method
+to change over time.
```

Comment:
```suggestion
Nebari is using a new way of distributing environments using [conda-store](https://github.com/quansight/conda-store). Since conda-store is under active development, be aware that the following details may change over time.
```